courses/views.py

```diff
@@ -124,3 +126,11 @@ class CourseRunViewSet(viewsets.ReadOnlyModelViewSet):
     """API for the CourseRun model"""
     serializer_class = CourseRunSerializer
     queryset = CourseRun.objects.all()
+
+
+class CatalogViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    """API for program/course catalog list"""
+    serializer_class = CatalogProgramSerializer
+    queryset = Program.objects.filter(live=True).prefetch_related(
+        "course_set__courserun_set", "programpage__thumbnail"
```

Comment:
I think this is a wrong field name. This should be `programpage__thumbnail_image`.